build.gradle

```diff
@@ -79,19 +147,27 @@ configure(subprojects.findAll { it.name.startsWith("maven-") }) {
     }
 }
 
-subprojects {
-    buildscript {
-        repositories {
-            google()
-            jcenter()
-            maven { url 'https://dl.bintray.com/kotlin/ktor' }
-            maven { url 'https://dl.bintray.com/kotlin/kotlin-eap' }
-        }
-        dependencies {
-            classpath "com.android.tools.build:gradle:3.4.1"
+
+if (build_snapshot_train) {
+    println "Hacking test tasks, removing stress and flaky tests"
+    allprojects {
+        tasks.withType(Test).all {
+            // Add: exclude '**/*TestName*' here
```

Comment:
So we do nothing? Or force configuring?

Reply:
We keep the configuration as-is for now, so it's easy to add flaky tests here in future